content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+|              | `--embed-runtime-image-tarball=PATH` |  The filename of a container to include in the package. Only for provider and function packages.                              |
+| `-e`         | `--examples-root="./examples"`       |  The path to a directory of examples related to the package.                               |
+|              | `--ignore=PATH,...`                  |  List of files and directories to ignore.                              |
+| `-o`         | `--package-file=PATH`                |  Directory and filename of the created package.                             |
+| `-f`         | `--package-root="."`                 |  Directory to search for YAML files.                              |
+{{< /table >}}
+
+The `crossplane xpkg build` command recursively looks in the directory set by 
+`--package-root` and attempts to combine any files ending in `.yml` or `.yaml` 
+into a package.
+
+All YAML files must be valid Kubernetes manifests with `apiVersion`, `kind`, 
+`metadata` and `spec` fields. 
+
+#### Ignore files
+
+Use `--ignore` to provide a list of files and directories to ignore.  
+
+For example,  
+`crossplane xpkg build --ignore="./test/*,kind-config.yaml"`
+
+#### Set the package name
+
+`crossplane` automatically names the new package a combination of the 
+`metadata.name` and a hash of the package contents and saves the contents 
+in the same location as `--package-root`. Define a specific location and 
+filename with `--package-file` or `-o`.  
+
+For example,  
+`crossplane xpkg build -o /home/crossplane/example.xpkg`.
+
+
+#### Include examples
+
+Include YAML files demonstrating how to use the package with `--examples-root`. 
+
+[Upbound Marketplace](https://marketplace.upbound.io/) uses files included with 
+`--examples-root` as documentation for published packages.
+
+#### Include a runtime image
+
+Functions and Providers require YAML files describing their dependencies and 
+settings as well as a container image for the function or provider controller.
+
+Using `--embed-runtime-image-name` downloads a specified container image and 
+includes the image inside the function or provider package. Use the 
+
+The `--embed-runtime-image-tarball` flag includes a local OCI image tarball 
+inside the function or provider package.
+
+
+### xpkg install
+
+Download and install Crossplane packages with  `crossplane xpkg install`.
+Crossplane packages.  
+
+By default the `crossplane xpkg install` command uses the Kubernetes 
+configuration defined in `~/.kube/config`.  
+
+Define a custom Kubernetes configuration file location with the environmental 
+variable `KUBECONFIG`.
+
+Specify the package kind, package file and optionally a name to give the package 
+inside Crossplane.
+
+`crossplane xpkg install <package-kind> <package-file> [<optional-name>`
+
+The package file must be an organization, image and tag on the 
+[Upbound Marketplace](https://marketplace.upbound.io/).
```

Comment:
This is only the default behaviour. You could specify `index.docker.io/crossplane-contrib/foo:v1.0.0` to install from a different registry.

Reply:
Testing this it looks like the default is always index.docker.io? I thought it was changed to be xpkg.upbound.io?

Reply:
I thought so too.

Reply:
Maybe I should open an issue. 

```shell
$ ./crank xpkg install provider upbound/provider-aws-s3/v0.42.0                                                                                           
provider/upbound-provider-aws-s3-v0-42-0 created

$ k describe provider/upbound-provider-aws-s3-v0-42-0 created
#--- snip ---#
  Warning  UnpackPackage  2s (x3 over 4s)  packages/provider.pkg.crossplane.io  cannot unpack package: failed to fetch package digest from remote: failed to fetch package descriptor with a GET request after a previous HEAD request failure: HEAD https://index.docker.io/v2/upbound/provider-aws-s3/v0.42.0/manifests/latest: unexpected status code 401 Unauthorized (HEAD responses have no body, use GET for details): GET https://index.docker.io/v2/upbound/provider-aws-s3/v0.42.0/manifests/latest: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:upbound/provider-aws-s3/v0.42.0 Type:repository]]
```

Reply:
Oh, my bad, install and update do default to index.docker.io.

This is because they're really just asking the package manager to install the thing, and the package manager defaults to index.docker.io. We considered changing that with this release so everything is consistent and not-confusing but we thought the change would be too risky/disruptive.